upgrade/UPGRADE-v9.2.0-dev.md

```diff
@@ -74,3 +74,11 @@ There you can find links to upgrade notes for other versions too.
         - `shopsys.doctrine.cache_driver.metadata_cache`
         - `shopsys.framework.cache_driver.annotations_cache`
         - `bestselling_product_cache`
+- update coding standards configuration ([#2415](https://github.com/shopsys/shopsys/pull/2415))
+    - version 8.3.48 of `symplify/easy-coding-standard` is now required
+    - removed dependency on `symplify/easy-coding-standard-tester`, you should change your custom coding standards tests
+        - you can find inspiration in https://github.com/shopsys/shopsys/pull/2415/files
+    - switch configuration of easy-coding-standard from yaml file to php file
+        - if you use default Shopsys Framework configuration, you can just use `ecs.php` (see #project-base-diff)
+        - for your custom configuration, you can leverage https://github.com/symplify/config-transformer
```

Comment:
ah, I was wondering whether you used it or not :grin: So there is a bug causing the comments are not converted? Shall we create an issue or a pull request? :slightly_smiling_face:

Reply:
comments are not converted by design see https://github.com/symplify/config-transformer#configuration
```
Note: Symfony YAML parser removes all comments, so be sure to go through files and add still-relevant comments manually.
```